docs/api-auth.md

```diff
@@ -25,30 +27,123 @@ There are three different ways a user is considered an admin:
+
+    ![Credentials Page](./img/credspage.png)
 
+1. Return to your command terminal and paste the auth code into the prompt.
 
+    ```
+    dce auth
+    ✔ Enter API Token: : █
+    ```
 
-### IAM Policy for DCE API requests
+1. You are now authenticated as a DCE User. Test that you have proper authorization by typing `dce leases list`.
+This will return an empty list indicating that there are currently no leases which you can view.
+If you are not properly authenticated as a user, you will see a permissions error.
 
-The IAM principal used to send requests to the DCE API must have sufficient permissions to execute API requests.
+    ```
+    dce leases list
+    []
+    ```
 
-The Terraform module in the repo provides an IAM policy with appropriate permissions for executing DCE API requests. The policy name and ARN are available as Terraform outputs.
+1. Users are not authorized to list child accounts in the accounts pool. Type `dce accounts list` to verify that you get a permissions error when trying to
+view information you do not have access to.
 
+    ```
+    dce accounts list
+    err:  [GET /accounts][403] getAccountsForbidden
+    ```
+
+1. You will need to be authenticated as an admin before continuing to the next section. Type `dce auth` to log in as a different user. Sign out, then enter the username
+and password for the admin that you created. As before, copy the auth code and paste it in the prompt in your command terminal.
+
+    ![Admin login](./img/quickstartadminlogin.png)
+
+1. Test that you have admin authorization by typing `dce accounts list`. You should see an empty list now instead of a permissions error.
+
+    ```
+    dce accounts list
+    []
+    ```
+
+## Using IAM Credentials
+
+The DCE API accepts authentication via IAM credentials using [SigV4 signed requests](https://docs.aws.amazon.com/general/latest/gr/sigv4_signing.html). 
+
+At minumum, the IAM credentials used to access DCE must have an attached policy that grants permission to invoke the DCE API Gateway methods, e.g.
+
+```
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "execute-api:Invoke"
+      ],
+      "Resource": [
+        "${aws_api_gateway_rest_api.gateway_api.execution_arn}/*"
+      ]
+    }
+  ]
+}
+JSON
+}
 ```
-terraform output api_access_policy_name
-terraform output api_access_policy_arn
-```
\ No newline at end of file
+
+This policy is accessible via the api_access_policy_name & api_access_policy_arn terraform outputs.
+
+Any requests made with IAM credentials that have sufficient permissions to invoke the DCE API, but which are not associated with a Congito User Pool User, will be treated as an `admin role <#admins>`_.
+
+The process for signing requests with SigV4 is somewhat involved, but luckily there are a number of tools to make this easier. For example:
```

Comment:
I think this page is getting a little confused, as it's mixing instructions for authenticating directly against the API together with directions for authenticating with DCE CLI.

If we're just talking DCE CLI, we don't need to talk about sig v4. But we _do_ need a section on where DCE CLI gets credentials from, and in what order. 
I guess I'd like to see this page re-org'd a bit, like

1. Configuring API User with AWS IAM
1. Configuring API Users with Cognito
1. Authenticating requests with the DCE CLI
1. Authenticating requests to the DCE API
1. Authorization Roles

what do you think?